docs/source/tutorials/lattice.ipynb

```diff
@@ -0,0 +1,872 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "b3f40e81",
+   "metadata": {},
+   "source": [
+    "## Quick Start Guide\n",
```

Comment:
the first line of a jupyter must be H1, namely `# Title`, you can move the quick start part below the title